packages/qgrid-ngx-plugins/src/lib/data-provider/models.ts

```diff
@@ -0,0 +1,13 @@
+import { GridModel } from '@qgrid/ngx';
```

Comment:
rename file to the server.ts or maybe page-server.ts and PageServer class? not sure

Reply:
Renamed interface to DataProviderPageServer